code/obj/mining.dm

```diff
@@ -566,6 +567,13 @@
 		if (src.active)
 			src.UpdateOverlays(src.active_overlay, "magnet_active")
 
+	// Sanity check to make sure we gib on no health
+	proc/check_should_die()
```

Comment:
should this send a forced update to the ui?

Reply:
it wouldnt do anything if it did